konflux.Containerfile

```diff
@@ -0,0 +1,45 @@
+FROM registry.access.redhat.com/ubi9/ubi:latest AS builder
+
+ARG FACT_TAG
+RUN if [[ "$FACT_TAG" == "" ]]; then >&2 echo "error: required FACT_TAG arg is unset"; exit 6; fi
```

Comment:
Since you don't use `FACT_TAG` in `builder` stage, you can remove these lines and add a check to the final stage.
```suggestion
```